src/psyclone/psyir/symbols/symbol.py

```diff
@@ -67,6 +188,8 @@ class Symbol(object):
     :param str name: name of the symbol.
     :param visibility: the visibility of the symbol.
     :type visibility: :py:class:`psyclone.psyir.symbols.Symbol.Visibility`
+    :param interface: the interface of the symbol.
```

Comment:
Please copy the same interface description as in the datasymbol docstring

Reply:
Oh, and say that by default is set to Local

Reply:
Updated.